server/postProcessHandler/Makefile

```diff
@@ -6,9 +6,9 @@ build-PostProcessHandler:
 	npm run fast-compile
 	npm run test
 	npm prune --production
-ifeq ($(OS),Windows_NT)
-	Xcopy /E * $(ARTIFACTS_DIR)
-else
-	cp -r dist "$(ARTIFACTS_DIR)/"
+# ifeq ($(OS),Windows_NT)
```

Comment:
You don't have to run this makefile if it is having issues. I wrote an equivalent PowerShell version if you need it.

Reply:
There were issues running it, but they've now been fixed